examples/vault-iam-auth/outputs.tf

```diff
@@ -0,0 +1,91 @@
+output "auth_client_public_ip" {
+  value = "${aws_instance.example_auth_to_vault.public_ip}"
+}
+
+output "auth_role_arn" {
+  value = "${aws_iam_role.example_instance_role.arn}"
+}
+
+output "asg_name_vault_cluster" {
+  value = "${module.vault_cluster.asg_name}"
+}
+
+output "launch_config_name_vault_cluster" {
+  value = "${module.vault_cluster.launch_config_name}"
+}
+
+output "iam_role_arn_vault_cluster" {
+  value = "${module.vault_cluster.iam_role_arn}"
+}
+
+output "iam_role_id_vault_cluster" {
+  value = "${module.vault_cluster.iam_role_id}"
+}
```

Comment:
There's some Gruntwork internal debate on this, but because `iam_role_id` is ambiguous on whether that refers to the IAM Role Name or ARN, I always prefer to explicitly name these `iam_role_name` or `iam_role_arn` to remove any ambiguity.

Reply:
That is not related to this PR, it would require refactoring `vault_cluster` and all other examples following your suggested pattern

Reply:
As per discussion on our zoom meeting, this would also be a breaking change.
For this reason we created an additional basecamp to-do categorised under a final project wrap-up to deal with this